scratchpad/tornado/test_tornado_io.py

```diff
@@ -6,7 +6,7 @@
 from tornado.util import ObjectDict
 
 from katcp import Sensor
-from tornado_io import *
+import tornado_io
```

Comment:
@ajoubertza is this script being used anywhere? i.e. Do we need it?

Reply:
The `scratchpad` directory is a place for experimental code.  This probably comes from the days when this package was being reworked to use tornado.  The file is not critical.